Assets/LeapMotion+OVR/Scripts/LeapCameraAlignment.cs

```diff
@@ -3,40 +3,50 @@
 using System.Collections.Generic;
 using Leap;
 
+/// <summary>
+/// Implements spatial alignment of cameras and synchronization with images
+/// </summary>
 public class LeapCameraAlignment : MonoBehaviour {
-  public static bool VerboseDebuging = false;
+  [SerializeField]
+  protected LeapImageRetriever imageRetriever;
+  [SerializeField]
+  protected HandController handController;
 
-  [HideInInspector]//[Range(0,1)]
+  [Header("Alignment Targets")]
+
+  [SerializeField]
+  protected Transform leftCamera;
+  [SerializeField]
+  protected Transform rightCamera;
+  [SerializeField]
+  protected Transform centerCamera;
+  public List<LeapImageBasedMaterial> warpedImages;
+
+  [Header("Alignment Settings")]
+
+  //[HideInInspector]
+  [Range(0,1)]
   public float tweenRewind = 0f;
-  [HideInInspector]//[Range(0,1)]
+  //[HideInInspector]
+  [Range(0,1)]
   public float tweenTimeWarp = 0f;
   [Range(0,2)]
   public float tweenPosition = 1f;
   [Range(0,2)]
   public float tweenForward = 1f;
-  
+
+  // Automatic Time Alignment
   public float latencySmoothing = 1f; //State delay in seconds
+  [HideInInspector]
   public SmoothedFloat frameLatency;
+  [HideInInspector]
   public SmoothedFloat imageLatency;
-
-  [Header("Alignment Targets")]
-  public LeapImageRetriever leftImages;
-  public LeapImageRetriever rightImages;
-  public HandController handController;
-  public List<LeapImageBasedMaterial> warpedImages;
   
-  [Header("Target History (micro-seconds)")]
-  public long maxLatency = 100000; //microseconds
-
-  LeapDeviceInfo deviceInfo;
-  private long timeFrame = 0;
-  private long lastFrame = 0;
-  private Vector3 virtualCameraStereo;
-  
-  //DEBUG
-  public KeyCode moreRewind = KeyCode.LeftArrow;
-  public KeyCode lessRewind = KeyCode.RightArrow;
+  // Manual Time Alignment
+  [HideInInspector]
   public float rewindAdjust = 1f; //Frame fraction
+  private KeyCode moreRewind = KeyCode.LeftArrow;
```

Comment:
I think it would be undesired behavior for the arrow keys to affect the rewind amount without the user knowing that it is the case.  Possibly make these fields public but default to KeyCode.None so that they can be enabled on a per-scene basis?